barman/output.py

```diff
@@ -853,6 +853,7 @@ def render_show_backup_base(backup_info, output_fun, header_row, nested_row):
         output_fun(nested_row.format("End Offset", backup_info["end_offset"]))
         output_fun(nested_row.format("Begin LSN", backup_info["begin_xlog"]))
         output_fun(nested_row.format("End LSN", backup_info["end_xlog"]))
+        output_fun(nested_row.format("Checksums", backup_info["data_checksums"]))
```

Comment:
We need to add this to the JSON output as well.

Reply:
Oh True!

Reply:
Besides that, I'm wondering that we might want to move this to the "outer section", i.e. have something like this:

```
Backup 20240726T231540:
  Backup Name            : incremental_3
  Server Name            : pg17
  System Id              : 7382227563826631291
  Status                 : DONE
  PostgreSQL Version     : 170000
  PGDATA directory       : /var/lib/pgsql/17/data
  Checksums            : on
```

I suggest that only so we are consistent with the changes that were agreed in the call for Andre's PR about `show-backup`.

Reply:
I applied this suggestion, but do you think it's worth considering adding a new section to this output? Considering that WAL Summarizer state will also come in another PR, maybe something like:

```
Backup 20240726T174335:
  Server Name            : pg17_server
  System Id              : 7376727553608684707
  Status                 : WAITING_FOR_WALS
  PostgreSQL Version     : 170000
  PGDATA directory       : /var/lib/pgsql/17/data

  Server Parameters:
      Checksums             :  on
      WAL Summarizer.   :  off  
    ....
    
```

I don't know. I don't think it's bad the way it is, just throwing some options on the table.

Reply:
I think it's worth talking about that with the team.
Personally, I like your suggestion.

For now let's proceed as it is. If we decide to do that the way you suggested, we can do that at once through Andre's PR as that one will be merged later.

Reply:
I like having it separately like suggested!

Reply:
Martin mentioned to me he also thinks this is good.